### PR TITLE
Jn direct control

### PR DESCRIPTION
**Pull request by Jay-Neighbours-I72**

### CODE REVIEW

File: src/config/locales/en.js [hunk 1]
```diff
@@ -4901,9 +4901,9 @@ const en = {
             acquired: 'Provide the date acquired',
             howAcquired: 'Provide how the financial interest was acquired',
             cost: 'Provide the cost (in U.S. dollars) at time of acquisition',
-            value: 'Provide the current value (in U.S. dollars) or the value at the time control or ownership was sold, lost or otherwise disposed of.',
-            relinquished: 'Provide the date control or ownership was relinquished.',
-            explanation: 'Provide explanation of how interest control or ownership was sold, lost or otherwise disposed of. '
+            value: 'Provide the current value (in U.S. dollars) or the value at the time control or ownership was sold, lost or otherwise disposed of',
+            relinquished: 'Provide the date control or ownership was relinquished',
+            explanation: 'Provide explanation of how interest control or ownership was sold, lost or otherwise disposed of '
```
Comment (bmallred) [inline src/config/locales/en.js]: @Jay-Neighbours-I72 we have a trailing whitespace for **explanation**... okay to remove?

File: src/config/locales/en.js [hunk 2]
```diff
@@ -4940,22 +4940,22 @@ const en = {
...
-              note: ''
+              title: 'Need help with date relinquished?',
+              message: 'Tell us when you were no longer in control or an owner of this foreign financial interest.',
+              note: 'Note: If you are not sure of the exact date give us your best guess and check the "Estimated" checkbox.'
             },
             explanation: {
-              title: 'Need help with this field?',
-              message: 'Provide explanation of how interest control or ownership was sold, lost or otherwise disposed of.',
+              title: 'Need help with this explanation?',
+              message: '',
```
Comment (bmallred) [inline src/config/locales/en.js]: @Jay-Neighbours-I72 okay to leave the message empty?

File: src/config/locales/en.js (hunk 1 above)
Comment (Jay-Neighbours-I72) [inline src/config/locales/en.js]: Yeah good to remove.

File: src/config/locales/en.js (hunk 2 above)
Comment (Jay-Neighbours-I72) [inline src/config/locales/en.js]: Good catch, I missed this. Will fix now

Comment (Jay-Neighbours-I72) [inline src/config/locales/en.js]: Fixed, should be good now @bmallred

**Pull request merged**